src/pm_kraken.jl

```diff
@@ -254,143 +232,81 @@ function writeenv(model::Kraken, tx::Vector{<:AcousticSource}, rx::AbstractArray
+      print_distance(io, [-location(rx1)[3] for rx1 ∈ rx] )  # 6)receiver depth(m)
+      nr = 1
+    elseif rx isa AcousticReceiverGrid2D
+      d = reverse(-rx.zrange)
+      if first(d) > last(d)
+       d = reverse(d)
+       zrev = true
       end
-      push!(rays, RayArrival(NaN64, NaN64, sb, bb, -deg2rad(aod), NaN64, raypath))
-    end
-  end
-  rays
-end
-
-function readarrivals(filename)
-  arrivals = RayArrival{Float64,Missing}[]
-  open(filename, "r") do io
-    s = strip(readline(io))
-    if occursin("2D", s)
-      f = parse(Float64, strip(readline(io)))
-      v = split(strip(readline(io)) ,r" +")
-      n = parse(Int, v[1])
-      txdepth = parse.(Float64, v[2:end])
-      n == length(txdepth) || error("Wrong number of txdepth entries in arrivals")
-      v = split(strip(readline(io)) ,r" +")
-      n = parse(Int, v[1])
-      rxdepth = parse.(Float64, v[2:end])
-      n == length(rxdepth) || error("Wrong number of rxdepth entries in arrivals")
-      v = split(strip(readline(io)) ,r" +")
-      n = parse(Int, v[1])
-      rxrange = parse.(Float64, v[2:end])
-      n == length(rxrange) || error("Wrong number of rxrange entries in arrivals")
-    else
-      v = split(s ,r" +")
-      f = parse(Float64, v[1])
-      n1, n2, n3 = parse.(Int, v[2:4])
-      txdepth = parse.(Float64, split(strip(readline(io)) ,r" +"))
-      rxdepth = parse.(Float64, split(strip(readline(io)) ,r" +"))
-      rxrange = parse.(Float64, split(strip(readline(io)) ,r" +"))
-      n1 == length(txdepth) || error("Wrong number of txdepth entries in arrivals")
-      n2 == length(rxdepth) || error("Wrong number of rxdepth entries in arrivals")
-      n3 == length(rxrange) || error("Wrong number of rxrange entries in arrivals")
-    end
-    for j ∈ 1:length(txdepth)
-      readline(io)
-      for k ∈ 1:length(rxdepth)
-        for m ∈ 1:length(rxrange)
-          count = parse(Int, strip(readline(io)))
-          for n ∈ 1:count
-            v = split(strip(readline(io)) ,r" +")
-            length(v) == 8 || error("Wrong number of data entries in arrivals")
-            A, ph, t, _, aod, aoa = parse.(Float64, v[1:6])
-            sb, bb = parse.(Int, v[7:8])
-            push!(arrivals, RayArrival(t, A * cis(deg2rad(ph)), sb, bb, -deg2rad(aod), deg2rad(aoa)))
-          end
-        end
+      r = rx.xrange ./ 1000.0
+      if first(r) > last(r)
+       r = reverse(r)
+       xrev = true
       end
+      print_distance(io, r)  # Receiver ranges (km)
+      print_distance(io, [-location(tx1)[3] for tx1 ∈ tx])  # Source depth (m)
+      print_distance(io, d)  # 6)receiver depth(m)
+      nr = length(d)
     end
+    print_distance(io, zeros(nr))# Number of receiver range displacement (same as NRD)
   end
-  sort(arrivals; by = a -> a.time)
 end
 
-function readshd(filename; xrev=false, zrev=false)
+
+
+function readshd_kraken(filename; xrev=false, zrev=false)
```

Comment:
We should probably namespace all the models with modules to avoid having to suffix `_kraken` eventually, but leave it in for now, and we can reorg when we reorg the entire repo.

Reply:
Noted.